superset-frontend/packages/superset-ui-core/src/utils/html.tsx

```diff
@@ -44,10 +44,26 @@ export function sanitizeHtml(htmlString: string) {
   return xssFilter.process(htmlString);
 }
 
+export function hasHtmlTagPattern(str: string): boolean {
```

Comment:
Do we have a package or a function inside our current dependencies that provide this feature?

Reply:
it doesn't seem so after quickly running through the package.json and checking with GPT, not sure if it would be as specific to the tags as this regex is

Reply:
seems https://cheerio.js.org/ would be the best fit, and is somewhere in our deptree, but at the same time I'd rather not bring yet another dependency for this relatively small/contained function...

Just for reference ->
```bash
$ npm ls cheerio
superset@0.0.0-dev /Users/max/code/superset/superset-frontend
└─┬ enzyme@3.11.0
  └── cheerio@1.0.0-rc.12
```

I think Enzyme is getting deprecated, so I wouldn't rely on this bycatch...

Reply:
Yeah that seems a little fishy. If it is a problem for complexity then we can just delete it. It was mostly a sort of triple check to expose any HTML so not the most useful of regex calls but I assume it will rarely be conjured.